pkg/scheduler/framework/plugins/apiinstalled/api_installed.go

```diff
@@ -36,7 +36,7 @@ func (p *APIInstalled) Name() string {
 // Filter checks if the API(CRD) of the resource is installed in the target cluster.
 func (p *APIInstalled) Filter(ctx context.Context, placement *policyv1alpha1.Placement, resource *workv1alpha2.ObjectReference, cluster *clusterv1alpha1.Cluster) *framework.Result {
 	if !helper.IsAPIEnabled(cluster.Status.APIEnablements, resource.APIVersion, resource.Kind) {
-		klog.V(2).Infof("cluster(%s) not fit as missing API(%s, kind=%s)", cluster.Name, resource.APIVersion, resource.Kind)
+		klog.V(2).Infof("cluster(%s) not fit as missing API(%s), kind(%s)", cluster.Name, resource.APIVersion, resource.Kind)
```

Comment:
klog.V(2).Infof("Cluster(%s) not fit as missing API(%s), kind(%s)", cluster.Name, resource.APIVersion, resource.Kind)